pkg/newlog/go.mod

```diff
@@ -1,14 +1,13 @@
 module github.com/lf-edge/eve/pkg/newlog
 
-go 1.23
-toolchain go1.24.1
+go 1.24
```

Comment:
I think it must be 1.24.1 due to security fixes....

Reply:
switched to 1.24.1 now, but for other [reasons](https://github.com/lf-edge/eve/pull/4702#issuecomment-2766936651)